### PR TITLE
Heroku and Amazon S3 configuration

### PR DESCRIPTION
**Pull request by houli**
Here's the Heroku and S3 config. Comment away(I'm sure there's something you'll want to say :smile:)
Example is running at http://agile-ocean-4713.herokuapp.com/

### CODE REVIEW

File: project/settings.py [hunk 1]
```diff
@@ -70,6 +62,16 @@
     'compressor.finders.CompressorFinder',
 )
 
+AWS_SECRET_ACCESS_KEY = os.environ['AWS_SECRET_ACCESS_KEY']
```
Comment (kevinwbaker) [inline project/settings.py:65]: Who's S3 bucket is Heroky configured to use?

File: project/settings.py [hunk 2]
```diff
@@ -70,6 +62,16 @@
...
+AWS_STORAGE_BUCKET_NAME = os.environ['AWS_STORAGE_BUCKET_NAME']
+BOTO_S3_BUCKET = AWS_STORAGE_BUCKET_NAME
+AWS_S3_SECURE_URLS = False
+STATICFILES_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
+S3_URL = 'http://%s.s3.amazonaws.com/' % AWS_STORAGE_BUCKET_NAME
+if not DEBUG:
+    STATIC_URL = S3_URL
+
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = "b2052d6e-ab87-45fb-a80e-83790c54ad037cac7022-7fa9-4fdc-808d-2d8faff813d70545bb68-6506-45e3-9e07-0393591be717"
```
Comment (kevinwbaker) [inline project/settings.py]: Could you move this out in a OS variable and then generate a new one. This value should never have been in a public repo like this.

File: project/settings.py (hunk 1 above)
Comment (houli) [inline project/settings.py:65]: My own at the minute.

**Comment by kevinwbaker** [comment]
Once you get the django `SECRET_KEY` into an OS variable this PR lgtm :turtle: 

Do you need me to merge it in or do you have permissions to do that?

**Comment by houli** [comment]
One thing this does change is development set up. There are a couple more steps necessary to get a development version of the site running locally and the README will have to be updated to account for that.

**Pull request merged**